database/youtube/data-structures.json

```
@@ -1,65 +1,74 @@
-[{
+[
+  {
     "name": "Jennys Lectures CSIT",
     "description": "This channel provides free courses for DSA, C, C++, Python etc.",
```

[Proofread] Always place commas where you're describing things in a list.
```suggestion
    "description": "This channel provides free courses for DSA, C, C++, Python, etc.",
```